src/msdo-interface.ts

```diff
@@ -0,0 +1,20 @@
+import { CommandType } from "./msdo-helpers";
+
+/*
+* Interface for the MicrosoftSecurityDevOps task
+*/
+export interface IMicrosoftSecurityDevOps {
+    readonly succeedOnError: boolean;

```

Comment:
NIT: right now we're not really honoring this property anywhere, might be worthwhile to just remove it to avoid confusion.